src/main/java/ch/njol/skript/effects/EffSendSignChange.java

```diff
@@ -0,0 +1,117 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import java.util.Arrays;
+
+import org.bukkit.DyeColor;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.SkriptColor;
+import ch.njol.util.Kleenean;
+
+@Name("Send Sign Change")
+@Description({
+	"Makes a player see a sign with text on it.",
+	"Color/Glowing is 1.17+ and usage of Skript chat options requires Paper spigot."
+})
+@Examples({
+	"set {_progress} to 100 - (100 - {progress::%uuid of player%})",
+	"if {_progress} is equal to 100:",
+		"\tset {_colour} to lime green",
+	"else if {_progress} is greater than 50:",
+		"\tset {_colour} to yellow",
+	"else if {_progress} is greater than 25:",
+		"\tset {_colour} to orange",
+	"else",
+		"\tset {_colour} to red",
+	"make player see sign at {signs::progress} with text \"Your progress is at %{_progress}%\" coloured {_colour}"
+})
```

Comment:
It's not required for the syntax to function.

Reply:
How come is that?! users will visit the docs and find the syntax support colors, when they try it on 1.16- it will says unknown syntax/etc. they will get confused, but because we have `@RequiredPlugins` annotation users will know why it's not working.

In addition to that, this is useful for other docs website to use.

Reply:
It's described in the description that the syntax has optional features. Just like EffSendBlockChange.

Reply:
Description is good but we still use Required Plugins annotation to mention that, to make it crystal clear what features does this syntax offer that requires specific versions/plugins, we always did that and it's a good habit.

Here is an existing example that you did yourself (LookAnchor optional feature)
![image](https://user-images.githubusercontent.com/20037329/233639360-df064230-ff3b-4327-a74d-9e034de72367.png)